gobblin-cluster/build.gradle

```diff
@@ -51,6 +51,7 @@ dependencies {
   compile externalDependency.helix
 
   runtime project(":gobblin-modules:gobblin-service-kafka")
+  runtime "org.apache.helix:helix-core:0.9.1"
```

Comment:
Let's drop this line. Let's keep the runtime version for OSS to 0.8.x.

Reply:
Sounds good, I realized later it still kept the issue as well